app_server/app/controllers/UserController.js

```diff
@@ -69,10 +95,21 @@ Class.logout = function (request, reply) {
   reply('Logged out!');
 };
 
+function clearUserPrivateInfo(user) {
+  if (user.password) {
+    delete user.password;
+  }
+  if (user.accessToken) {
+    delete user.accessToken;
+  }
```

Comment:
I'm not entirely sure on this either, but is there any purpose in checking if the properties exist before deleting them?
I tried testing a little, but the additional check seems to be irrelevant.
(it may cause trouble in strict mode if `user` isn't an object, but that would happen here too)